controller/src/beerocks/master/db/db.cpp

```diff
@@ -964,7 +964,7 @@ std::set<std::string> db::get_node_siblings(const std::string &mac, int type)
 
     auto parent = get_node(n->parent_mac);
     if (!parent) {
-        LOG(ERROR) << "parent for node " << mac << " does not exist";
+        LOG(WARNING) << "parent for node " << mac << " does not exist";
```

Comment:
Please explain in the commit message why you changed to WARNING. I really don't get what's the difference.. this commit can be dropped I recon.

Reply:
Its printed every time 1905_topology_response arrives (its a lot) or errors in the code .
The reason is that GW's al_mac  fails to retrieve a parent and we get this error (we apply this on every al_mac  on the neighbor list ).
I didn't want to fix this issue  as part of my PR, but i'm sure this is not an error message that why i added this commit (i can add this explanation to the PR if you want )